Remove two elements that are not available in illumos termios.h

--- sbcl-2.6.4/contrib/sb-posix/constants.lisp.orig	2026-04-29 21:27:06.000000000 +0200
+++ sbcl-2.6.4/contrib/sb-posix/constants.lisp	2026-05-02 22:48:31.150774474 +0200
@@ -433,9 +433,7 @@
               (tcflag-t oflag "tcflag_t" "c_oflag")
               (tcflag-t cflag "tcflag_t" "c_cflag")
               (tcflag-t lflag "tcflag_t" "c_lflag")
-              ((array cc-t) cc "cc_t" "c_cc")
-              (speed-t ispeed "speed_t" "c_ispeed")
-              (speed-t ospeed "speed_t" "c_ospeed")))
+              ((array cc-t) cc "cc_t" "c_cc")))
 
  ;; utime(), utimes()
  #-win32
--- sbcl-2.6.4/contrib/sb-posix/interface.lisp.orig	2026-04-29 21:27:06.000000000 +0200
+++ sbcl-2.6.4/contrib/sb-posix/interface.lisp	2026-05-04 17:30:35.021138552 +0200
@@ -822,11 +822,7 @@
    (lflag :initarg :lflag :accessor termios-lflag
           :documentation "Local modes.")
    (cc :initarg :cc :accessor termios-cc :array-length nccs
-       :documentation "Control characters.")
-   (ispeed :initarg :ispeed :accessor termios-ispeed
-           :documentation "Input speed.")
-   (ospeed :initarg :ospeed :accessor termios-ospeed
-           :documentation "Output speed."))
+       :documentation "Control characters."))
   (:documentation
    "Instances of this class represent I/O characteristics of the terminal."))
 
